### PR TITLE
Pre-request 32 callbacks of each method

### PR DESCRIPTION
**Pull request by vjpai**
Before this we were only registering 1 of each method
Longer-term we should vary this dynamically.

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE                                         FILE SIZE
 ++++++++++++++ GROWING                           ++++++++++++++

 -------------- SHRINKING                         --------------
  -0.0%     -32 [None]                            -2.23Ki  -0.0%
  -0.0%     -16 src/cpp/server/server_cc.cc           -16  -0.0%
      -7.1%    -124 grpc::Server::Server                 -124  -7.1%
      -0.7%      -5 [Unmapped]                             -5  -0.7%

  -0.0%     -48 TOTAL                             -2.25Ki  -0.0%



```

File: src/cpp/server/server_cc.cc [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 #include <grpc/grpc.h>
 #include <grpc/support/alloc.h>
+#include <grpc/support/cpu.h>
```
Comment (AspirinSJL) [inline src/cpp/server/server_cc.cc]: This is not used in the PR?

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 2,019,390      Total (=)      2,019,390

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
11,180,027      Total (<)     11,180,036

 No significant differences in binary sizes


```

File: src/cpp/server/server_cc.cc (hunk 1 above)
Comment (vjpai) [reply src/cpp/server/server_cc.cc]: Oops! I went a different route. I'll remove this.

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 2,019,390      Total (=)      2,019,390

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
11,180,036      Total (>)     11,180,035

 No significant differences in binary sizes


```

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE                                         FILE SIZE
 ++++++++++++++ GROWING                           ++++++++++++++

 -------------- SHRINKING                         --------------
  -0.0%     -32 [None]                            -2.23Ki  -0.0%
  -0.0%     -16 src/cpp/server/server_cc.cc           -16  -0.0%
      -7.1%    -124 grpc::Server::Server                 -124  -7.1%
      -0.7%      -5 [Unmapped]                             -5  -0.7%

  -0.0%     -48 TOTAL                             -2.25Ki  -0.0%



```

**Comment by grpc-testing** [comment]
```
Corrupt JSON data (indicates timeout or crash): 
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.new: 10
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.old: 10


[microbenchmarks] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 2,019,040      Total (=)      2,019,040

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
11,179,735      Total (=)     11,179,735

 No significant differences in binary sizes


```

**Comment by grpc-testing** [comment]
```
Corrupt JSON data (indicates timeout or crash): 
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.new: 10
    bm_call_create.BM_IsolatedFilter_ClientChannelFilter_NoOp_.counters.old: 10


[microbenchmarks] No significant performance differences
```

**Pull request merged**